includes/settings.php

```diff
@@ -492,19 +492,18 @@ public function get_errors()
 
 		if (count($this->errors))
 		{
-			$user = $this->get_user();
 			foreach ($this->errors as $error)
 			{
 				if (is_array($error))
 				{
```

Comment:
```suggestion
				if ($error)
				{
```

Reply:
`$error` may not always be an array. But it needs to be converted to one if it isn't. So we need to check if it, and process it as one if it is, and if not, then process it into one.